### PR TITLE
Add route for 17 Corolla

### PR DESCRIPTION
**Pull request by sshane**

### CODE REVIEW

File: selfdrive/test/test_routes.py [hunk 1]
```diff
@@ -251,20 +251,11 @@
...
-  "b0f5a01cf604185c|2018-01-26--10-54-38": {
-    'carFingerprint': TOYOTA.COROLLA,
-    'enableCamera': True,
-    'enableDsu': False,
-  },
-  "b0f5a01cf604185c|2018-01-26--10-59-31": {
-    'carFingerprint': TOYOTA.COROLLA,
-    'enableCamera': False,
-    'enableDsu': False,
+    'enableGasInterceptor': True,
```
Comment (adeebshihadeh) [inline selfdrive/test/test_routes.py:258]: no need for this, it's not used anymore

File: selfdrive/test/test_routes.py [hunk 2]
```diff
@@ -251,20 +251,11 @@
     'enableCamera': True,
     'enableDsu': False,
   },
-  "b0f5a01cf604185c|2018-01-26--00-54-32": {
+  "4e45c89c38e8ec4d|2021-05-02--02-49-28--15": {
```
Comment (adeebshihadeh) [inline selfdrive/test/test_routes.py:254]: remove the segment

File: selfdrive/test/test_routes.py (hunk 1 above)
Comment (ShaneSmiskol) [reply selfdrive/test/test_routes.py:258]: Should enableDsu be false since I've unplugged mine?

Comment (adeebshihadeh) [reply selfdrive/test/test_routes.py]: you can just leave out everything besides carFingerprint

**Comment by adeebshihadeh** [comment]
Can you add an additional check to this line so routes with a missing enableCamera are also included? https://github.com/commaai/openpilot/blob/master/selfdrive/test/test_models.py#L22

**Comment by ShaneSmiskol** [comment]
Assuming `enableCamera` is True if it's missing? Got it. It might be nicer to switch to a base class for the routes and then define the defaults in the init function.

**Comment by adeebshihadeh** [comment]
Yes. No need for a class, I'll clean this up soon.

**Comment by ShaneSmiskol** [comment]
Oh whoops I already went ahead and started to refactor test_models. I can revert if you want to do it yourself, but I'll push to let you see.

File: selfdrive/test/test_routes.py [hunk 3]
```diff
@@ -35,473 +35,130 @@
...
+  def __init__(self, route, car_fingerprint, *, enable_camera=True, enable_dsu=False, enable_gas_interceptor=False):
+    self.route = route
+    self.car_fingerprint = car_fingerprint
+    self.enable_camera = enable_camera
+    self.enable_dsu = enable_dsu
+    self.enable_gas_interceptor = enable_gas_interceptor
+
+
+routes = [
+  TestRoute("420a8e183f1aed48|2020-03-05--07-15-29", CHRYSLER.PACIFICA_2017_HYBRID),
```
Comment (adeebshihadeh) [inline selfdrive/test/test_routes.py:49]: the other info was only for some old test we don't run anymore. we only need the route and car now, so let's revert these changes

Comment (ShaneSmiskol) [reply selfdrive/test/test_routes.py:49]: No problem

Comment (ShaneSmiskol) [reply selfdrive/test/test_routes.py]: Not sure why one of the routes is failing now...

Comment (adeebshihadeh) [reply selfdrive/test/test_routes.py]: that's unrelated, it's a recent process replay regression

Comment (ShaneSmiskol) [reply selfdrive/test/test_routes.py]: Thought so. Let me know if you want me to make a PR to clean up the routes dict
